rogue-thi-app/components/cards/CalendarCard.js

```diff
@@ -26,7 +26,7 @@ export default function CalendarCard () {
       try {
         exams = (await loadExamList())
           .filter(x => !!x.date) // remove exams without a date
-          .map(x => ({ name: `Prüfung ${x.titel}`, begin: x.date }))
+          .map(x => ({ name: `${t('calendar.exam')} ${x.titel}`, begin: x.date }))
```

Comment:
Please add the additional parameter `t`` to the React hook in line 46. [router, t]